app/src/main/java/com/framgia/fsalon/utils/Utils.java

```diff
@@ -3,6 +3,7 @@
 import android.support.design.internal.BottomNavigationItemView;
 import android.support.design.internal.BottomNavigationMenuView;
 import android.support.design.widget.BottomNavigationView;
+import android.text.TextUtils;
```

Comment:
ERROR: Unused import: android.text.TextUtils. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports